homeassistant/components/fan/wemo.py

```diff
@@ -0,0 +1,303 @@
+"""
+Support for WeMo humidifier.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/fan.wemo/
+"""
+import asyncio
+import logging
+from datetime import timedelta
+import requests
```

Comment:
Add a blank line between standard library and 3rd party imports.

Reply:
Just to clarify, is this what you're looking for?

```python
import asyncio
import logging
from datetime import timedelta
import requests

import async_timeout
import voluptuous as vol
import homeassistant.helpers.config_validation as cv

from homeassistant.components.fan import (
    DOMAIN, SUPPORT_SET_SPEED, FanEntity,
    SPEED_OFF, SPEED_LOW, SPEED_MEDIUM, SPEED_HIGH)
from homeassistant.exceptions import PlatformNotReady
from homeassistant.const import ATTR_ENTITY_ID
```